index.html

```diff
@@ -11568,6 +11569,11 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>The <code>aria-haspopup</code> property is an enumerated type. <a>User agents</a> MUST treat any value of <code>aria-haspopup</code> that is not included in the list of allowed values, including an empty string, as if the value <code>false</code> had been provided. To provide backward compatibility with ARIA 1.0 content, user agents MUST treat an <code>aria-haspopup</code> value of <code>true</code> as equivalent to a value of <code>menu</code>.</p>
 				<p><a>Assistive technologies</a> SHOULD NOT expose the <code>aria-haspopup</code> property if it has a value of <code>false</code>.</p>
 				<p class="note">A <rref>tooltip</rref> is not considered to be a popup in this context.</p>
+				<p class="note"><code>aria-haspopup</code> is most relevant to use when there is a visual indicator in the element that triggers the popup. 
+					For example, many controls styled with a downward pointing triangle or chevron, or three consecutive dots, have become standard visual indicators that a popup will display when the control is activated. 
```

Comment:
```suggestion
					For example, many controls styled with a downward pointing triangle, chevron, or ellipsis (three consecutive dots) have become standard visual indicators that a popup will display when the control is activated. 
```

Reply:
Would it be appropriate to link to this portion of the ellipsis Wikipedia entry? https://en.wikipedia.org/wiki/Ellipsis#Usage_in_computer_system_menus

From the page: 

> In computer menu functions or buttons, an ellipsis means that upon selection more options (sometimes in the form of a dialog box) will be displayed, where the user can or must make a choice.

Reply:
Surprisingly, the Wikipedia entry for chevron does not have a similar section.

Reply:
prefer not to link to wikipedia - not sure we need the definition